rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1162,7 +1169,12 @@ def _load_model(
         model_dir: Text,
         finetune_mode: bool = False,
     ) -> "RasaModel":
-        file_name = meta.get("file")
+        if "file" not in meta:
+            raise CorruptMetaDataException(
+                f"Metadata for trained component {cls.name}"
```

Comment:
```suggestion
                f"Metadata for trained component {cls.name} "
```

Reply:
Think we need this space (or one on the line below), else the message will read: `"Metadata for trained component {cls.name}is missing the 'file' attribute."`